src/main/java/bio/terra/service/profile/ProfileApiController.java

```diff
@@ -96,4 +101,15 @@ protected void initBinder(final WebDataBinder binder) {
         BillingProfileModel profileModel = profileService.getProfileById(id, user);
         return new ResponseEntity<>(profileModel, HttpStatus.OK);
     }
+
+    @Override
+    public ResponseEntity<PolicyResponse> addProfilePolicyMember(
+        @PathVariable("id") String id,
+        @PathVariable("policyName") String policyName,
+        @Valid @RequestBody PolicyMemberRequest policyMember) {
```

Comment:
doesn't the valid tag expect a validation file? or is it totally generic?

Reply:
Good catch. I have not really understood how @Valid was supposed to work. I added the binder for this validation check and also added the tags for validation of the offset and limit on enumerate.